react/components/AccordionFilterItem.js

```diff
@@ -80,6 +85,16 @@ const AccordionFilterItem = ({
               <span className={`${handles.accordionFilterItemIcon} fr`}>
                 <IconCaret orientation="down" size={10} />
               </span>
+              {showAppliedFiltersOverview && quantitySelected > 0 && (
+                <div
+                  className={classNames(
+                    handles.accordionSelectedFilters,
+                    'f6 c-action-primary'
```

Comment:
If we can't click on the element, it doesn't make sense to use `c-action-primary`

See: https://styleguide.vtex.com/#/Styles?id=section-colors